config/jobs/image-pushing/k8s-staging-csi-secrets-store.yaml

```diff
@@ -0,0 +1,46 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated
+  # for each repo.
+  kubernetes-sigs/secrets-store-csi-driver:
+    # The name should be changed to match the repo name above
+    - name: secrets-store-csi-driver-push-image
+      # TODO: move this to a more appropriate cluster.
+      cluster: test-infra-trusted
```

Comment:
@claudiubelu should we use `k8s-infra-prow-build-trusted`?

Reply:
I've written this in the same way I did for ``k8s-staging-e2e-test-images.yaml``, but looking at the other yaml files, it seems that has been updated to ``k8s-infra-prow-build-trusted``.